src/deserialize.ts

```diff
@@ -123,7 +123,7 @@ export default function deserialize(node: MdastNode, opts?: OptionType) {
         type: types.image,
         children: [{ text: '' }],
         [imageSourceKey]: node.url,
-        [imageCaptionKey]: node.alt,
+        [imageCaptionKey]: [{ text: node.alt }],
```

Comment:
Personally don't like that these two are diverging, I also wonder if this should be a flag or could be more generic so consumers could configure.

Thoughts?

Reply:
+1 to making this configurable as per my comment on issue #44, I think that would solve my scenario nicely.